core/drivers/gic.c

```diff
@@ -858,6 +888,20 @@ void gic_dump_state(void)
 	}
 }
 
+void gic_spi_get_prop(size_t it, uint8_t *cpu_mask, uint8_t *prio)
```

Comment:
I want to get property before requesting the IRQ so that it can be used later in the irq_release flow. I will remove this function if the cpu_mask is cleared and the priority set to default value.